src/test/resources/CaseDetailsWithNoHearings.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
I think this file is the same as the `CaseDataArray.json` file right? Or did you want to create separate files so we explicitly know that it is a case with no hearings? I don't mind either way, just thought i'd raise it.

Reply:
if the content is exactly the same we should resue it.

Reply:
The test files should be named properly so that it can be reused.

Reply:
the content is different @jackmaloney .